runtime/onert/backend/trix/DevContext.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_BACKEND_TRIX_DEV_CONTEXT_H__
+#define __ONERT_BACKEND_TRIX_DEV_CONTEXT_H__
+
+#include <libnpuhost.h>
+
+namespace onert
+{
+namespace backend
+{
+namespace trix
+{
+
+class DevContext
+{
+public:
+  DevContext()
+  {
+    auto device_count = getNPUdeviceByType(NPUCOND_TRIV2_CONN_SOCIP);
```

Comment:
maybe getnumNPUdeviceByType ?

Reply:
@jyoungyun Please use `getnumNPUdeviceByType` as @chunseoklee mentioned. : )

Reply:
> maybe getnumNPUdeviceByType ?
> @jyoungyun Please use getnumNPUdeviceByType as @chunseoklee mentioned. : )

Yes, that's what I was trying to exactly. Thank you!